_automating-configurations/api/search-workflow-state.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Search for a workflow state
+parent: Workflow APIs
+nav_order: 65
+---
+
+# Search for a workflow
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/flow-framework/issues/475).    
+{: .warning}
+
+You can search for resources created by workflows using a query matching a field. The fields you can search correspond to those returned by the [Get Workflow Status API]({{site.url}}{{site.baseurl}}/automating-configurations/api/get-workflow-status/).
```

Comment:
The meaning of the first sentence is slightly unclear. Do we mean something like "You can search for resources created by workflows by matching a query to a field"?

Reply:
Yes, basic search behavior. I like your wording though!